test/Microsoft.Azure.SignalR.Tests.Common/TestRestClient.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Net;
+using Microsoft.Azure.SignalR.Tests.Common;
+
+namespace Microsoft.Azure.SignalR.Tests
+{
+    public class TestRestClient : SignalRServiceRestClient
```

Comment:
can be a `private sealed` inner class of `HealthApiFacts`

Reply:
This class will be used by ServiceApi in the future.

Reply:
I thought they would be in the same class and also the same test class